roles/cinder-common/templates/etc/sudoers.d/cinder

```diff
@@ -1 +1,6 @@
+{% if os == 'ubuntu' %}
```

Comment:
I think this is an artifact of it coming from source/giftwrap vs. Ubuntu or RedHat